ngxtop/config_parser.py

```diff
@@ -59,17 +61,21 @@ def get_access_logs(config):
     access_log = Literal("access_log") + ZeroOrMore(parameter) + semicolon
     access_log.ignore(pythonStyleComment)
 
+    access_logs_dict = {}
     for directive in access_log.searchString(config).asList():
         path = directive[1]
         if path == 'off' or path.startswith('syslog:'):
             # nothing to process here
             continue
-
-        format_name = 'combined'
+        access_logs_dict[path] = ['combined']
```

Comment:
you can see this line : log_formats_dict = dict(get_log_formats(config_str)) in function 
detect_log_config . There did not lose all previously gathered format info for path